client/lib/feed-stream-store/index.js

```diff
@@ -338,7 +321,7 @@ export default function feedStoreFactory( storeId ) {
 		store = new FeedStream( {
 			id: storeId,
 			fetcher: wpcomUndoc.readFollowing.bind( wpcomUndoc ),
-			keyMaker: feedKeyMaker,
+			keyMaker: mixedKeyMaker,
```

Comment:
we probably need a limiter for fields here now too for updates, like on site streams.

Otherwise we build the whole post in the update set over and over again on the server.

Reply:
see getStoreForSite.

we want that same limiter for update fetches on the feed stream too. Unsure about gap fetches.